beacon-chain/core/helpers/committee.go

```diff
@@ -304,6 +293,7 @@ func AttestationParticipants(
 			participants = append(participants, validatorIndex)
 		}
 	}
+	sort.Slice(participants, func(i, j int) bool { return participants[i] < participants[j] })
```

Comment:
can you add a test assertion confirming it is sorted ? you can add it to one of the current tests

Reply:
the current tests failed and i had to fix their order to be sorted . is it enough?

Reply:
its good in that case